### DISCUSSION TITLE
how to get a Access Token

### DISCUSSION THREAD

**Discussion by zaze06** [spotify-web-api-java/spotify-web-api-java#318] [Q&A] [answered]
Hi, I wonder how I can get a user Access Token to get what song I'm playing or use the skip song. but I can't get it to work, so how am I supposed to get it to work? is it even possible to do?

**Comment by dargmuesli** [comment, marked as answer]
Please read the README: https://github.com/spotify-web-api-java/spotify-web-api-java#authorization

**Reply by zaze06** [reply]
Hi, thanks for this. I have read the code but I'm unsure what I should put as the `code` for the `authorizationCodeRequest` Sorry if I just misunderstood something

**Reply by dargmuesli** [reply]
That's what `authorizationCodeUriRequest.execute()` provides. Again, that's described in detail at https://developer.spotify.com/documentation/general/guides/authorization/code-flow/

**Reply by zaze06** [reply]
so the URI I'm given from `authorizationCodeUriRequest.execute()` should be open in my browser? and then I get a code from that if I understand it correctly?

**Reply by zaze06** [reply]
Got it to work. thanks

**Reply by zaze06** [reply]
The solution was that when opening the web page to get the code when I was redirected I took the code from the new webpage that it opened and input it into my program.
